src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  background-color: #000;
+  padding: 0 12px;
+}
+
+.first,
+.third,
+.second {
+  background-color: #ddd;
+  margin: 0;
+}
```

Comment:
One more time:
Classes `first`, `second` and `third` belong to three different elements and have the same styles.
You can use just single class instead of these three.